### PR TITLE
feat: support issue assign via `/assign`, fix typo

### PR DESCRIPTION
**Pull request by unconsolable**
Signed-off-by: unconsolable <chenzhipeng2012@gmail.com>

Close datafuselabs/databend#2890

What is changed:
- Add `/assign` command support
- Fix a typo

### CODE REVIEW

**Comment by unconsolable** [comment]
@BohuTANG @ariesdevil PTAL

File: actions/issue_action.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func (s *IssueAction) DoAction(event interface{}) error {
 		body := event.Comment.Body
 		log.Infof("Issue comments: %+v , %+v coming", event.Sender.Login, body)
 		switch body := strings.ToLower(body); {
-		case strings.HasPrefix(body, "/assignme"):
+		case strings.HasPrefix(body, "/assignme"), strings.HasPrefix(body, "/assign"):
```
Comment (Xuanwo) [inline actions/issue_action.go:44]: It looks like we don't need to check `/assignme` if we use `HasPrefix` here, as `/assign` covers `/assignme`.

Comment (Xuanwo) [reply actions/issue_action.go:44]: Will we support `/assign @xxx` in the further?

Comment (unconsolable) [reply actions/issue_action.go]: I try to add support for `/assign @xxx`, PTAL.

Comment (BohuTANG) [reply actions/issue_action.go]: Sorry for the later, I missed this notification.
Thank you @Xuanwo for review it, this patch looks good to me

**Pull request merged**